docs/source/developers/bug_reports.rst

```diff
@@ -186,8 +173,8 @@ Arrow is an expansive project supporting many languages and organized into a
 number of components. Identifying the affected component(s) helps new issues
 get attention from appropriate contributors.
 
-* Use the **Component** field to indicate the area of the project that your
-  issue pertains to (for example "Python" or "C++").
+* Use a **Component** label to indicate the area of the project that your
+  issue pertains to (for example "Component: Python" or "Component: C++").
```

Comment:
It's certainly useful to document this (also for committers), but at the same time the wording above is not directly applicable in general for people opening issues, as they can't "use" (add) a label to issues themselves.

Reply:
Ah, I thought anybody can select the component from the label list. In that case we can write something like: "Component label, which can be added by a committer of the Apache Arrow project, is used to indicate the area of the project that your issue pertains to ..."